cloud/amazon/sts_session_token.py

```diff
@@ -75,9 +75,9 @@
 
 # Use the session token obtained above to tag an instance in account 123456789012
 ec2_tag:
-  aws_access_key: "{{ session_credentials.sts_creds.access_key }}"
-  aws_secret_key: "{{ session_credentials.sts_creds.secret_key }}"
-  security_token: "{{ session_credentials.sts_creds.session_token }}"
+  aws_access_key: "{{ session_credentials.credentials.access_key_id }}"
```

Comment:
This gets out of sync with https://github.com/ansible/ansible-modules-extras/blob/devel/cloud/amazon/sts_assume_role.py#L77